lib/Minify/ScssCssSource.php

```diff
@@ -0,0 +1,175 @@
+<?php
+use Leafo\ScssPhp\Compiler;
+use Leafo\ScssPhp\Server;
+use Leafo\ScssPhp\Version;
+
+/**
+ * Class for using SCSS files
+ *
+ * @link https://github.com/leafo/scssphp/
+ */
+class Minify_ScssCssSource extends Minify_Source {
+    /**
+     * @var Minify_CacheInterface
+     */
+    private $cache;
+
+    /**
+     * Parsed SCSS cache object
+     *
+     * @var array
+     */
+    private $parsed;
+
+    /**
+     * @inheritdoc
+     */
+    public function __construct(array $spec, Minify_CacheInterface $cache)
+    {
+        parent::__construct($spec);
+
+        $this->cache = $cache;
+    }
+
+    /**
+     * Get last modified of all parsed files
+     *
+     * @return int
+     */
+    public function getLastModified()
+    {
+        $cache = $this->getCache();
+
+        return $cache['updated'];
+    }
+
+    /**
+     * Get content
+     *
+     * @return string
+     */
+    public function getContent()
+    {
+        $cache = $this->getCache();
+
+        return $cache['content'];
+    }
+
+    /**
+     * Make a unique cache id for for this source.
```

Comment:
"unique" amounts to common string + md5 hash these days?
Either provide some actual uniqueness (via UUID or similar) or drop the word unique